curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-intermediate-algorithmic-thinking-by-building-a-dice-game/649f0e6e081b80023d1b4fab.md

```diff
@@ -0,0 +1,286 @@
+---
+id: 649f0e6e081b80023d1b4fab
+title: Step 15
+challengeType: 0
+dashedName: step-15
+---
+
+# --description--
+
+Add a `click` event listener on the `const` named `rollDiceBtn` which basically targets the `roll-dice-btn` id element. And then pass a es6 arrow function as the second argument.
+
+# --hints--
+
+You should add a `click` event listener on the `const` named `rollDiceBtn`.
```

Comment:
Should probably split into two tests - one to check for the `click` event, the second to check for the arrow function.